services/venice-controller/src/main/java/com/linkedin/venice/controller/server/AdminTopicMetadataRoutes.java

```diff
@@ -120,11 +121,15 @@ public Route updateAdminOperationProtocolVersion(Admin admin) {
         AdminSparkServer.validateParams(request, UPDATE_ADMIN_OPERATION_PROTOCOL_VERSION.getParams(), admin);
         String clusterName = request.queryParams(CLUSTER);
         Long adminOperationProtocolVersion = Long.parseLong(request.queryParams(ADMIN_OPERATION_PROTOCOL_VERSION));
+        AdminTopicMetadataGrpcResponse internalResponse = requestHandler.updateAdminOperationProtocolVersion(
+            UpdateAdminOperationProtocolVersionGrpcRequest.newBuilder()
+                .setClusterName(clusterName)
+                .setAdminOperationProtocolVersion(adminOperationProtocolVersion)
+                .build());
```

Comment:
This is new to me. I don't find the builder function somehow. Do we need to proactively close it? I guess no, since it's not a thread and should be GC automatically? cc: @sushantmane

Reply:
`UpdateAdminOperationProtocolVersionGrpcRequest` is generate based on `gRPC` proto definition by protobuf compiler. We don't need to close it as it's like any other regular java clas

Reply:
Adding to @sushantmane , this is the [PR](https://github.com/linkedin/venice/pull/1533) which we define the proto rpc. Then at compile time, it will generate the java classes based on the definition.